src/blocks/stopwatch.scss

```diff
@@ -0,0 +1,43 @@
+.stopwatch {
+  position: relative;
+  @include circleSize($size-stop-watch);
+
+  background: $bg-stop-watch;
+  
+  &__minut::before {
+    content: "";
+    @include positArrow($width-minut);
+    @include paramArrow(
+      $width-minut, 
+      $hight-minut, 
+      $bg-minut, 
+      $anime-minut
+    );
+  }
+
+  &__sekond::after {
+    content: "";
+    @include positArrow($width-second);
+    @include paramArrow(
+      $width-second, 
+      $hight-second, 
+      $bg-second, 
+      $anime-second
+    );
+  }
+
+  &__center {
+    position: absolute;
+    @extend %position-center;
+    transform: translate(-50%, -50%);
```

Comment:
You can include `transform: translate(-50%, -50%)` in the extend.
It is standard way to center the element, so `top` , `left` and `transform` should be used together.

Reply:
Thanks for the review ))